examples/api/lib/cupertino/date_picker/cupertino_timer_picker.0.dart

```diff
@@ -66,39 +64,34 @@ class _TimerPickerExampleState extends State<TimerPickerExample> {
           fontSize: 22.0,
         ),
         child: Center(
-          child: Column(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: <Widget>[
-              _TimerPickerItem(
-                children: <Widget>[
+          child: Column(mainAxisAlignment: MainAxisAlignment.center, children:
+              <Widget>[
+                _TimerPickerItem(children: <Widget>[
```

Comment:
This looks also strange, why is the argument list of Column not switching to tall style with trailing comma if it doesn't fit on one line?

Reply:
I think it's the same as the previous example where the line comments below are throwing it off.

Reply:
Agreed. That the formatter is trying so hard to accomudate a comment feels odd to me. The comment line length is the easiest to adjust, so I kinda wish it would format the code nicely while almost ignoring the length of the comment lines so I can then go in to adjust those line length. The workflow right now is kinda wonky: I have to know that the comments may be throwing off the formatting and then go in to adjust them without knowing to what line length I have to adjust them to. After adjusting them, I'd ran the formatter again to see if it produces something better now and keep iterating until I've guessed the best line length for the comment. THe other way around would be less steps.

Reply:
FWIW, I've completely removed the comments here as well and I am still getting the same odd formatting.

Reply:
> I kinda wish it would format the code nicely while almost ignoring the length of the comment lines so I can then go in to adjust those lines.

Yeah, the problem is that it's more complex than that. In many cases a line has both a comment and other code, and the other code *can* be formatted so that the comment can be made to not overflow and that's what the user wants.

So figuring out how to reliably ascribe a "cause" to the overflow becomes, I think, non-trivial. But I haven't really had the time to sit down and think through an actual model for how the formatter could reason about overflow in a more sophisticated way. I just know it's unlikely to be simple. :)